CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+# Community Code of Conduct
```

Comment:
Why do we add this own code of conduct, when Eclipse already has a code of conduct: https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php

How do we make sure not to diverge from the Eclipse CoC?

Last part is, that adding a code of conduct is a new addition and not in the official scope of "beta release note". The release note should include release notes and version information and should not add additional content. This would have to be added by an independent PR or at least being mentioned in the description of the PR.